sdk/python/feast/infra/feature_servers/multicloud/Dockerfile

```diff
@@ -9,8 +9,8 @@ RUN apt update && \
 
 RUN pip install pip --upgrade
 COPY . .
+RUN pip install ".feast[aws,gcp,snowflake,redis,go,mysql]"
```

Comment:
Do you need the period as well as `feast`?

Reply:
good catch